src/platform/posix/posix_ipcconn.c

```diff
@@ -363,7 +363,7 @@ ipc_peerid(ipc_conn *c, uint64_t *euid, uint64_t *egid, uint64_t *prid,
 	*prid = uc.pid;
 	*znid = (uint64_t) -1;
 	return (0);
-#elif defined(NNG_HAVE_LOCALPEERCRED)
+#elif defined(NNG_HAVE_LOCALPEERCRED) && !defined(__FreeBSD__)
```

Comment:
FreeBSD does some to have a libc getpeereid() call that might be superior here anyway.